docs/eventlisteners.md

```diff
@@ -241,9 +241,11 @@ spec:
 
 CEL interceptors parse expressions to filter requests based on JSON bodies and
 request headers, using the [CEL](https://github.com/google/cel-go) expression
-language.
+language. Please read the [cel-spec language definition](https://github.com/google/cel-spec/blob/master/doc/langdef.md)
+for more details on the expression language syntax.
 
-Supported features include case-insensitive matching on request headers.
+In addition to the standard CEL expression language syntax, additional 
```

Comment:
Can we add a link to cel_expressions.md doc here?

Reply:
done